server/game/stateWatchers/CardsLeftPlayThisPhaseWatcher.ts

```diff
@@ -0,0 +1,42 @@
+import { Card } from '../core/card/Card';
+import { UnitCard } from '../core/card/CardTypes';
+import { UpgradeCard } from '../core/card/UpgradeCard';
+import { StateWatcherName } from '../core/Constants';
+import Player from '../core/Player';
+import { StateWatcher } from '../core/stateWatcher/StateWatcher';
+import { StateWatcherRegistrar } from '../core/stateWatcher/StateWatcherRegistrar';
+
+export interface CardLeftPlayEntry {
+    card: UnitCard | UpgradeCard;
```

Comment:
We have the type union `InPlayCard` that covers this, just needs to be updated so it is exported